tools/tsp-client/src/commands.ts

```diff
@@ -412,3 +413,82 @@ export async function sortSwaggerCommand(argv: any): Promise<void> {
   await writeFile(swaggerFile, JSON.stringify(sorted, null, 2));
   Logger.info(`${swaggerFile} has been sorted.`);
 }
+
+export async function combineSwaggersCommand(argv: any): Promise<void> {
+    Logger.info("Combining multiple swagger files into a single file...");
+    const outputDir = argv["output-dir"];
+    const swaggerFiles = argv["swagger-files"];
+    const swaggerFileCount = swaggerFiles?.length ?? 0;
+    if (swaggerFileCount <= 1) {
+        throw new Error(`2 or more swagger files must be specified, ${swaggerFileCount} provided.`);
+    }
+
+    await ensureDirectory(outputDir);
+    
+    const documents = [];
+    for (const file of swaggerFiles) {
+        if (!(await doesFileExist(file))) {
```

Comment:
We should specify somewhere that this command only accepts the path to the swagger file and that we do not accept urls to a swagger since we do accept both in some commands.